plugins/woocommerce/src/RestApi/Routes/V4/PaymentGateways/Controller.php

```
@@ -0,0 +1,432 @@
+			return new WP_Error( 'rest_setting_value_invalid', __( 'An invalid setting value was passed.', 'woocommerce' ), array( 'status' => 400 ) );
+		}
+	}
+
+	/**
+	 * Validate checkbox based settings.
+	 *
+	 * @param string $value  Field value.
+	 * @param array  $setting Setting data.
+	 * @return string
+	 */
+	public function validate_setting_checkbox_field( string $value, array $setting ): string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$setting'. (undefined)

(UnusedFormalParameter)